packages/block-editor/src/components/inserter/style.scss

```diff
@@ -777,3 +777,9 @@ $block-inserter-tabs-height: 44px;
 		}
 	}
 }
+
+:focus {
```

Comment:
I believe it's just for debugging and not meant for merging 😅?

Reply:
Yes it is. I switched the PR to draft, just in case.